client/src/store/global.js

```diff
@@ -25,16 +25,42 @@ function standardJsonInit(method, body) {
 }
 
 function useGlobal() {
-  const [userState, setUserState] = useState({})
-  const [eventState, setEventState] = useState({})
-  const [eventInviteeState, setEventInviteeState] = useState ({})
-  const [groupState, setGroupState] = useState({})
-  const [exerciseState, setExerciseState] = useState({})
-  const [exerciseResponseState, setExerciseResponseState] = useState({})
-
-  
+  const [userState, setUserState] = useState(
```

Comment:
We should comment out, incase it becomes useful

Reply:
Leave everything that is in global.js. Query functions are now working properly but haven't been tested throughly. As well as the testing suite for query functions, seed, and the user routes fixed how a user is deleted